Jenkinsfile

```diff
@@ -27,13 +39,17 @@ def jobMatrix(String node_type, String ctestcmd, List specs, Closure callback) {
         try {
           checkout scm
 
+          if (isLegacyChange()) {
+            ctestcmd = ctestcmd + ' -DBUILD_METHOD=legacy'
+          }
+
```

Comment:
Could you look into moving that outside the jobMatrix function?
Maybe after the `def ctestcmd = "ctest -VV -S FairSoft_test.cmake"`?

Reply:
Not sure how, it needs to run after the scm checkout and this only happens in node context. I could write it inside the lambda on the callsite.

Reply:
Ohh, if it needs the scm checkout, then all is fine. Maybe add a quick comment to that effect here?